cmd/hypper/hypper.go

```diff
@@ -114,3 +114,18 @@ func loadReleasesInMemory(actionConfig *action.Configuration) {
 	// Must reset namespace to the proper one
 	mem.SetNamespace(settings.Namespace())
 }
+
+// This re-Inits the config so we can set the Storage Driver namespace to what is obtained
+// from the chart.
+// This is because on helm you can only set the namespace via the --namespace flag which gets into
+// the envsettings.namespace and the helm.go loads that var into the Configuration object during
+// cobra.OnInitialize. To maintain compativility we do the same but then during the command execution
+// we read the chart annotations and change the release namespace on the fly. This is ok for the
+// namespace install but the Storage Driver, which is already loaded, has a different namespace (usually default)
+// and the Release gets stored in the wrong namespace (stored *for* helm, for k8s is installed properly)
+// This results into a disconnect between where the release is installed and where hypper/helm looks in the
+// Storage Driver to search for it. With this below we re-Init the configuration and hance the Storage Driver to
+// have the namespace correctly set in sync with the Release
+func ReinitConfigForNamespace(cfg *action.Configuration, namespace string, logger log.Logger) {
+	_ = cfg.Init(settings.RESTClientGetter(), namespace, os.Getenv("HELM_DRIVER"), logger.Debugf)
```

Comment:
Reading env variables outside environment.go... I do not really like that, can we move that to a settings variable?

Just saw this is also down when assigning the `helmDriver` variable in line 65 and in `cmd/hypper/list.go`.....

Reply:
well, I was trying to mimick the first `Init` which is done just above in line 65. This is how helm does it, so Im guessing we just lift it from there?

Reply:
I wonder why they dont directly use `settings` here to set both of those options...maybe an oversight??

Reply:
updated to use settings as it turns out that we can add it there with no issues that I can see. I wonder if we should upstream this and the other env var used...